bedrock/firefox/templates/firefox/hello/index-b.html

```diff
@@ -2,7 +2,7 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
-{% extends "firefox/hello/index-2016.html" %}
+{% extends "firefox/hello/index.html" %}
 
 {% set_lang_files "firefox/hello/index-2016" %}
```

Comment:
Yes, `firefox/hello-2016.lang` is the [one we have](https://l10n.mozilla-community.org/~pascalc/langchecker/?locale=all&website=0&file=firefox/hello-2016.lang).